plugins/imdb.py

```diff
@@ -49,15 +47,23 @@ def imdb_url(match, bot):
     if imdb_id == "":
         imdb_id = match.group(4).split('/')[-2]
 
-    params = {'i': imdb_id}
-    request = requests.get("http://www.omdbapi.com/", params=params, headers=headers)
+    params = {'id': imdb_id}
+    request = requests.get(
+        "https://imdb-scraper.herokuapp.com/title",
+        params=params,
+        headers=headers)
     content = request.json()
 
-    if content['Response'] == 'True':
-        out = '\x02%(Title)s\x02 (%(Year)s) (%(Genre)s): %(Plot)s'
-        if content['Runtime'] != 'N/A':
-            out += ' \x02%(Runtime)s\x02.'
-        if content['imdbRating'] != 'N/A' and content['imdbVotes'] != 'N/A':
-            out += ' \x02%(imdbRating)s/10\x02 with \x02%(imdbVotes)s\x02' \
-                   ' votes.'
-        return out % content
+    if content['success'] == True:
```

Comment:
This could most likely be simplified to `if content['success']:`, or `if content['success'] is True` if the value should specifically be `True` and not just evaluate to `True`.